### ISSUE TITLE
Fix so that the characters are inside the box, making it more readable.

### ISSUE THREAD

**Issue by a19karwa** [HGustavs/LenaSYS#10067]
When putting out a relation element, the name currently goes outside the box. It should be fixed so all of the characters stays inside the rect element.

![image](https://user-images.githubusercontent.com/81557429/114849204-d282fe00-9ddf-11eb-86ed-e29e63c952f1.png)

**Comment by a19hammi** [comment]
fixed in #11057
